### DISCUSSION TITLE
[ENG-269] Migrate Self Hosted Infisical to Paid Cloud Hosted Infisical

### DISCUSSION THREAD

**Discussion by alberk8** [Infisical/infisical#5714] [Feature Request]
### Feature description

A clear and concise description of what the feature should be.
Provide a migration from running self hosting to paid cloud hosting.

### Why would it be useful?

Why would this feature be useful for Infisical users?
This allow for certain deployment to offer offline and online options.

<sub>From [SyncLinear.com](https://synclinear.com) | [ENG-269](https://linear.app/infisical/issue/ENG-269/migrate-self-hosted-infisical-to-paid-cloud-hosted-infisical)</sub>

**Comment by vmatsiiako** [comment]
@alberk8 if you or anyone else needs it, please reach out to support@infisical.com and we'd be able to assist with that.
